raster-foundry/raster-foundry
# Fix bringing up development environment from scratch

## Overview

Ran into https://github.com/azavea/operations/issues/214, so I decided to upgrade the development environment to use Ubuntu 16.04. This also includes a shift to the `ansible_local` Vagrant provisioner over the default (remote) `ansible` provisioner.

A handful of unused scripts and modules were moved in the process.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~


~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
~- [ ] Any new SQL strings have tests~

## Testing Instructions

Destroy your local development environment and recreate it using this branch (or, clone this branch somewhere else on your workstation and spin it up from scratch there). Follow the steps in the `README` to ensure that things work as expected.

I got an error citing that I was trying to create a user with a non-existent system user. I suspect this is because one of the Auth0 variables in the default `.env` file is out-of-date?
